main.js

```diff
@@ -1,15 +1,15 @@
 const btnSave=document.querySelector('#btnSave');
-let notes="";
+let notes;
 init();
 
 function init(){
-    let out="";
+    let out;
     let noteArray=JSON.parse(localStorage.getItem('noteData'));
     
     if(noteArray != null && noteArray != ""){
         noteArray= JSON.parse(localStorage.getItem('noteData'));
         
-        for (let x=0; x < noteArray.length; x++){
+        for(let x=0; x < noteArray.length; x++){
             out += "<option value=" +x + ">";
             out += noteArray[x].title;
             out += "</option>";
```

Comment:
To jest ta część o DOM-ie, można o wiele przyjemniej go edytować, ale wszystko na spokojnie 😄